pkg/util/etcdutil/migrate.go

```diff
@@ -119,16 +121,20 @@ func createMigratedEtcdCluster(httpcli *http.Client, host, podIP string) error {
   }
 }`, podIP))
 
-	resp, err := httpcli.Post(
-		fmt.Sprintf("%s/apis/coreos.com/v1/namespaces/kube-system/etcdclusters", host),
-		"application/json", bytes.NewReader(b))
-	if err != nil {
-		return err
+	uri := "/apis/coreos.com/v1/namespaces/kube-system/etcdclusters"
+	res := restclient.Post().RequestURI(uri).Body(b).Do()
+	// Where do i put "application/json" ?
+	//fmt.Sprintf("%s/apis/coreos.com/v1/namespaces/kube-system/etcdclusters", host),"application/json", bytes.NewReader(b))
```

Comment:
Oh sorry, I mis-read this for some reason. We should be fine without explicitly setting the content type to json